sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -599,8 +600,14 @@ def receive_message(self, **kwargs: Any) -> Optional[QueueMessage]:
             process_storage_error(error)
 
     @distributed_trace
-    def receive_messages(self, **kwargs):
-        # type: (Any) -> ItemPaged[QueueMessage]
+    def receive_messages(
+            self,
+            *,
+            messages_per_page: Optional[int] = None,
+            visibility_timeout: Optional[int] = None,
+            max_messages: Optional[int] = None,
```

Comment:
nit: This is offscreen, but can you move the docstring for `max_messages` above `timeout`. I like to have all of the listed keywords above what's in `kwargs`